src/presets/seegno.json

```diff
@@ -96,49 +44,5 @@
     "while",
     "with"
   ],
-  "requireParenthesesAroundIIFE": true,
-  "requireSemicolons": true,
-  "requireShouldAssertionExecution": true,
-  "requireSpaceAfterBinaryOperators": true,
-  "requireSpaceAfterComma": true,
-  "requireSpaceAfterKeywords": [
-    "case",
-    "catch",
-    "do",
-    "finally",
-    "for",
-    "if",
-    "return",
-    "switch",
-    "try",
-    "typeof",
-    "while",
-    "with"
-  ],
-  "requireSpaceAfterLineComment": true,
-  "requireSpaceBeforeBinaryOperators": true,
-  "requireSpaceBeforeBlockStatements": true,
-  "requireSpaceBeforeDestructuredValues": true,
```

Comment:
Can't find this in the description of https://github.com/seegno/eslint-config-seegno/pull/45.